src/tests/utils/insert-data.ts

```diff
@@ -3,7 +3,13 @@ import users from '../test-data/users.json';
 import userSessions from '../test-data/user-sessions.json';
 import notes from '../test-data/notes.json';
 import noteSettings from '../test-data/notes-settings.json';
+// import user_sessions from '../test-data/userSessions.json';
+import {  updateRefreshTokenExpiry } from './insert-tomorrow-expiration-day';
 
+const filePath = 'src/tests/test-data/user-sessions.json';
```

Comment:
you don't need to write future session to user-sessions.json file on each tests run. Rather than that I'd suggest altering `insertUserSessions()` function in such a way that it also inserts 1 additional note with dynamically formed date

Reply:
sorry I don't clearly understand what do you want. Would you like to I delete _insert-tomorrow-expiration-day.ts_ and rewrite it's functionality in _insert-data.ts_ ?